.ci/test

```diff
@@ -69,14 +69,12 @@ function test_with_coverage() {
 
 ###############################################################################
 
-if [[ "${SKIP_UNIT_TESTS}" != "" ]]; then
-  echo ">>>>>Skipping unit tests"
-else
-  echo ">>>>> Invoking unit tests"
-  TEST_PACKAGES="pkg"
-  GINKGO_COMMON_FLAGS="-r -timeout=1h0m0s --randomizeAllSpecs --randomizeSuites --failOnPending  --progress"
-  test_with_coverage
-  echo ">>>>> Finished executing unit tests"
-fi
+ if [[ "${SKIP_INTEGRATION_TESTS}" != "" ]]; then
+   echo ">>>>> Skipping integration tests"
+ else
+   echo ">>>>> Invoking intergration tests"
+   .ci/pipeline_integration_test
+   echo ">>>>> Finished executing integration tests"
+ fi
 
-echo "CI tests have passed successfully"
+ echo "CI tests have passed successfully"
```

Comment:
I think it would be better to keep the unit tests running code , but keeping it commented. So that in final merge you don't forget.